soc/sim.py

```diff
@@ -137,6 +139,7 @@ def main():
         build=args.run,
         run=args.run,
         sim_config=sim_config,
+        interactive=False,
```

Comment:
Does this prevent interactive sim (`make PLATFORM=sim load`)?   Do we need to make this configurable?